src/uct/ib/ud/base/ud_iface.c

```diff
@@ -846,6 +846,11 @@ void uct_ud_iface_release_desc(uct_recv_desc_t *self, void *desc)
 void uct_ud_iface_handle_failure(uct_ib_iface_t *iface, void *arg,
                                  ucs_status_t status)
 {
+    if (status != UCS_ERR_ENDPOINT_TIMEOUT) {
+        /* Local side failure - treat as fatal */
+        ucs_fatal("local error: %s", ucs_status_string(status));
```

Comment:
would it print the error CQE with HW status ?

Reply:
Then need to overload this function for ud_mlx5, will update